themes/digital.gov/layouts/shortcodes/do-dont-row.html

```diff
@@ -0,0 +1,10 @@
+{{- if and (.Get "do") (.Get "dont") -}}
+  <tr scope="row">
+    <td data-label="Do">
+      {{- .Get "do" -}}
+    </td>
+    <td data-label="Don't">
+      {{- .Get "do" -}}
```

Comment:
This is the same value as `Do`.

```suggestion
      {{- .Get "dont" -}}
```

Reply:
Doh! Good catch ;)